platform/mds/mds/src/main/java/org/motechproject/mds/config/ModuleSettings.java

```diff
@@ -215,6 +242,7 @@ public void serialize(ModuleSettings value, JsonGenerator jgen, SerializerProvid
             jgen.writeObjectField("emptyTrash", value.isEmptyTrash());
             jgen.writeObjectField("timeValue", value.getTimeValue());
             jgen.writeObjectField("timeUnit", value.getTimeUnit());
+            jgen.writeObjectField("gridSize", value.getGridSize());
```

Comment:
off topic, but what's the reason for all this? why isn't this class just a pojo, but stores everything in a map instead?